sqlmesh/core/table_diff.py

```diff
@@ -277,6 +277,20 @@ def name(e: exp.Expression) -> str:
                         1,
                         0,
                     ).as_("row_joined"),
+                    exp.func(
+                        "IF",
+                        exp.and_(
```

Comment:
why is this an and?

shouldn't it be a null grain if any part of the key is null?

Reply:
Right, that's a mistake I'll remove the `and_`